### DISCUSSION TITLE
Scan Changed Files Only

### DISCUSSION THREAD

**Discussion by girlpunk** [oxsecurity/megalinter#2892] [Q&A] [answered]
I'd like to add the Vale linter to a project, however as we had no previous style guide set up there are several thousand errors from it.

Sure I saw an option for this before, but I can't find it now. Is it possible to have Megalinter only lint files that have been changed as part of a pull request? That way, existing files won't trigger errors until someone changes them (and can fix the errors at the same time).

Thanks!

**Comment by echoix** [comment, marked as answer]
Look at `VALIDATE_ALL_CODEBASE` in http://megalinter.io/latest/config-variables/
